### PR TITLE
Small text updates to converter readme

### PR DESCRIPTION
**Pull request by MichaelClifford**
Removed references to old repo name "locallm". And added a few other text edits.

### CODE REVIEW

File: convert_models/README.md [hunk 1]
```diff
@@ -33,12 +33,12 @@ streamlit run convert_models/ui.py
...
 
-This process writes the models into a Podman volume under a `gguf/` directory and not directly back to the user's host machine (This could be changed in an upcoming update if it is required).
+This process writes the models into a podman volume under a `gguf/` directory and not directly back to the user's host machine (This could be changed in an upcoming update if it is required).
 
-If a user wants to access these models to use with the llamacpp model-service, they would simply point their model-service volume mount to the Podman volume created here. For example:
+If a user wants to access these models to use with the llamacpp_python model server, they would simply point their model service to the correct podman volume at run time. For example:
 
 ```
-podman run -it -p 8001:8001 -v models:/locallm/models:Z -e MODEL_PATH=models/gguf/<MODEL_NAME> -e HOST=0.0.0.0 -e PORT=8001 llamacppserver
+podman run -it -p 8001:8001 -v models:/locallm/models:Z -e MODEL_PATH=models/gguf/<MODEL_NAME> -e HOST=0.0.0.0 -e PORT=8001 llamacpp_python
```
Comment (Gregory-Pereira) [inline convert_models/README.md]: the standard model path has also changed

**Review by Gregory-Pereira — COMMENTED** [review]
/lgtm but 1 nit

File: convert_models/README.md (hunk 1 above)
Comment (MichaelClifford) [reply convert_models/README.md]: good catch, fixing

Comment (MichaelClifford) [reply convert_models/README.md]: `/opt/app-root/src/converter/converted_models/gguf/` is this the new default path?

Comment (Gregory-Pereira) [reply convert_models/README.md:41]: for llamacpp its `/locallm/models/granite-7b-lab-Q4_K_M.gguf` Makefiles are the source of truth on these things

Comment (Gregory-Pereira) [reply convert_models/README.md]: Yeah AFAIK that is the right path

**Review by rhatdan — APPROVED** [review]
LGTM

**Pull request merged**